perses/app/relative_setup.py

```diff
@@ -689,7 +693,7 @@ def _generate_vacuum_topologies(self, topology_proposal, old_positions):
 
         return ligand_topology_proposal, old_ligand_positions
 
-    def _solvate_system(self, topology, positions, model='tip3p',phase='complex', box_dimensions=None):
+    def _solvate_system(self, topology, positions, model='tip3p',phase='complex', box_dimensions=None,ionic_strength=0.15*unit.molar):
```

Comment:
probably best not to hardcode the conc here, but instead reference to `self._ionic_strength`?